### DISCUSSION TITLE
Cannot override cli inspection severity for inspection not model related

### DISCUSSION THREAD

**Discussion by geemorin** [structurizr/cli#167] [Q&A] [answered]
### Expected

For example: if I set 

```
workspace {
  model {
    properties {
      structurizr.inspection.views.* warning
      structurizr.inspection.workspace.* warning
    }
  }
}
```

I expect to see there errors as warnings, not as ERRORS

### Actual

```
ERROR  | workspace.scope               | This workspace has no defined scope. It is recommended that the workspace scope is set to "Landscape" or "SoftwareSystem".
ERROR  | views.view.key                | The view with key "SystemLandscape-001" has an automatically generated view key and this is not guaranteed to be stable over time.
ERROR  | views.styles.element.metadata | The element style for tag "sf-start" has metadata hidden, which may introduce ambiguity on rendered diagrams.
ERROR  | views.styles.element.metadata | The element style for tag "sf-end" has metadata hidden, which may introduce ambiguity on rendered diagrams.
```

### Steps to reproduce

Create a basic workspace with the properties above. Run the docker cli inspection with 

inspect -w <workspace.dsl> -severity error

### Version/build information

docker pull structurizr/cli:2024.12.07

### Severity

Minor

### Priority

I have no budget and there's no rush, please fix this for free

### More information

Thank you!

**Comment by simonbrowndotje** [comment, marked as answer]
From the [docs](https://docs.structurizr.com/workspaces/inspections):

> The severity of model related violations can also be changed by properties at the model or workspace level, and view related violations can be changed by properties at the viewset or workspace level

I think you just need to move the workspace property to the workspace level, and the view property to the view level:

```
workspace {

    properties {
        structurizr.inspection.workspace.* warning
    }

    model {

    }

    views {
        properties {
            structurizr.inspection.views.* warning
        }
    }

}
```

Alternatively you can move everything to the workspace level. If that doesn't work, please provide a full DSL example.

**Comment by geemorin** [comment]
Oh its got to be it thank you!
